test/OpenTelemetry.Instrumentation.Http.Tests/OpenTelemetry.Instrumentation.Http.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>$(SupportedNetTargets)</TargetFrameworks>
+    <TargetFrameworks>net9.0;$(SupportedNetTargets)</TargetFrameworks>
```

Comment:
I think this change can be removed after Sean's PR https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/2313 merges.